src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -347,7 +350,7 @@ class ContentSidebar extends PureComponent<Props, State> {
      * @param {Error} e - API error
      * @return {void}
      */
-    fetchVersionsErrorCallback = (e: Error) => {
+    fetchVersionsErrorCallback = (e: $AxiosXHR<any>) => {
```

Comment:
Make sure to update this error prop when it gets passed down to ActivityFeed.js via `activityFeedError`